### PR TITLE
Bump fastboot

### PR DESCRIPTION
**Pull request by lucacorti**
I have tested that bumping `fastboot` and `fastboot-express-middleware` fixes the issue I'm experiencing with `ember-fetch`, `fastboot-app-server` and `ember-data@3.9` reported here ember-fastboot/ember-cli-fastboot/issues/686.

### CODE REVIEW

**Comment by lucacorti** [comment]
I aligned the travis test matrix to bring it in line with current fastboot node releases. Also, `fastboot-express-middleware` needs a release with bumped fastboot I think.

**Comment by snewcomer** [comment]
@lucacorti This looks good!  Is a new release of fastboot-express-middleware required before merging this?

**Comment by lucacorti** [comment]
@snewcomer I don't think a release of `fastboot-express-middleware` is strictly needed, this is sufficient, but would be nice to get rid of the fastboot 1.x dependecy.

**Comment by SergeAstapov** [comment]
@kratiahuja could this be merged and new version released if possible, please?

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 language: node_js
 node_js:
-  - "4"
+  - "stable"
   - "6"
```
Comment (locks) [inline .travis.yml]:
```suggestion
```

**Comment by kratiahuja** [comment]
@SergeAstapov  I am very sorry for the delay. I am happy to merge this PR. cc: @locks  not sure I understand your suggestion, but the diff looks fine to me. Please let me know if I am ready this incorrectly.

**Comment by kratiahuja** [comment]
Thank you @lucacorti and apologies for the delay again. I'll get a release out tonight cc @rwjblue

**Comment by lucacorti** [comment]
@kratiahuja Great, thanks! Would be great if you cpuld also roll a new release for `fastboot-express-middleware` and bump that dependency too.

**Comment by locks** [comment]
@kratiahuja I was removing Node 6 since it's no longer a supported version, but it can be done afterwards!

**Comment by kratiahuja** [comment]
@locks apologies for that, I'll didn't realize it. I'll send in a PR for the same tomorrow. 

@lucacorti Sure let me look into that after releasing this.

**Comment by snewcomer** [comment]
🎉 Yayyy!  FYI #105 is ready to go that drops 6.  Thank you again!

**Comment by kratiahuja** [comment]
Sorry for the delay, I have a PR for bumping fastboot-express-middleware: https://github.com/ember-fastboot/fastboot-app-server/pull/109, once the tests pass for that, I'll merge and release fastboot-app-server@2.0.0

**Comment by kratiahuja** [comment]
`fastboot-app-server@2.0.0` is published 🎉

**Pull request merged**